### PR TITLE
[SMP-2228] Update SMP monitoring for 0.10.0

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update SMP monitoring for 0.10.0

For reviewers, a preview is available.

[Monitoring options](https://654107faefdcf313c4a813d5--harness-developer.netlify.app/docs/self-managed-enterprise-edition/monitor-harness-on-prem)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 1]
```diff
@@ -275,11 +275,11 @@ Now you can add a dashboard to view metrics via query.
 
 Here are some sample open source dashboards:
```
Comment (sarthak-harnessio) [inline docs/self-managed-enterprise-edition/monitor-harness-on-prem.md:276]: Hi, can we mention that these are some custom dashboards provided by harness from our public repository? Fits better to the context now that we are shipping our own dashboards.

**Comment by sarthak-harnessio** [comment]
Looks good! Thanks @brian-f-harness

**Comment by sarthak-harnessio** [comment]
Looks good! Thanks @brian-f-harness

**Review by sarthak-harnessio — DISMISSED** [review]

File: docs/self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 2]
```diff
@@ -271,16 +271,53 @@ Follow the below steps on your Kubernetes cluster to deploy Grafana:
...
+- HTTP Response 95th percentile
+- HTTP Error Count
+- JVM Total Memory Max
+- Committed & Used
+- JVM Non-Heap Committed & Used
+- JVM Metaspace Memory Pool Used & Committed
+- JVM Threads Blocked / Deadlock State
+- DW Threads Utilization
+
+You can view the metrics by importing the System Observability Dashboard in our [Harness Dashboards repository](https://github.com/harness/harness-dashboards/).
```
Comment (sarthak-harnessio) [inline docs/self-managed-enterprise-edition/monitor-harness-on-prem.md:315]: Hi, can we also add the link to System Observability Dashboard, it's actually under another folder which might be hard to locate in the dashboards repo. 
https://github.com/harness/harness-dashboards/blob/main/System_Health/System_Observability_Dashboard.json

**Review by kapilgarg1996 — DISMISSED** [review]

**Review by sarthak-harnessio — APPROVED** [review]
lgtm!

**Review by sarthak-harnessio — APPROVED** [review]

**Review by pratmit — APPROVED** [review]

**Pull request merged**